tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "rootDir": "." /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */,
     "strict": true /* Enable all strict type-checking options. */,
     "noImplicitAny": true /* Raise error on expressions and declarations with an implied 'any' type. */,
-    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+    "esModuleInterop": true, /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+    "moduleResolution": "node"
```

Comment:
`vercel/ncc` wanted me to add a `moduleResolution` unsure if this is right 😅

Reply:
`node` is the default for commonjs so this should be a noop 👍